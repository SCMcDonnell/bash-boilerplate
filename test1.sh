#!/bin/bash

source boilerplate.sh

assert_number_of_arguments 1  
