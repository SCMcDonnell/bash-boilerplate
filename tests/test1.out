Usage: test1.sh
Expected 1 arguments (got 0)
